kolla/node_custom_config/blazar.conf

```diff
@@ -58,6 +58,7 @@ enable_polling_monitor = true
 
 [virtual:floatingip]
 billrate = {{ blazar_floatingip_billrate }}
+enable_polling_monitor = true
```

Comment:
What is the default behavior if this is not set?

Reply:
If `enable_polling_monitor` is False for FIPs, then it won't use the FIP monitor plugin to heal resources ( in this case FIPs)

Reply:
Right, but what if it's not set at all?
Just want to be sure that blazar will start successfully if an older config is in use.

Reply:
The monitor is set to run periodically and whenever it tries to run it fails with following error
```oslo_config.cfg.NoSuchOptError: no such option enable_notification_monitor in group [virtual:floatingip]```